README.md

```diff
@@ -27,6 +27,37 @@ Applying 2 changes ........ OK
 Done ...................... use `git diff/add/commit/etc`
 ```
 
+## `generate` command
+
+The `generate` command accepts a description of the file(s) needed and a set of files which are used to generate a new file based on their contents.
+
+Here's a breakdown of the generate process:
+1. Format the user's request along with the necessary files using a generate template that will be "autocompleted" by OpenAI.
+1. Have OpenAI attempt to complete the prompt and retrieve OpenAI's response.
+1. If OpenAI succeeded, parse the response and extract the newly generated files.
+1. Write the generated files either to the disk or to STDOUT. 
+
+### Usage 
+
+```sh
+# to create a Jupyter Notebook which uses a GPU to accelerate machine-learning tasks
+copilot-ops generate --write \
+	--request "create a Jupyter Notebook that specifies a GPU resource"
```

Comment:
We can try to touch up the README next.
I think we can remove the --write from these examples - we don't want it to run wet on people's first encounter.
We can also compact the run examples because shell strings do not require eol escaping (\)
```bash
copilot-ops generate -r '
  create a Jupyter Notebook that specifies a GPU resource
  ...
'
```

Reply:
Oh I never realized, I was always under the impression that you needed a long `<<EOF` statement for a heredoc.